### PR TITLE
Support sub-group URL

### PR DESCRIPTION
**Pull request by hiroponz**
Support sub-group URL (e.g. https://gitlab.com/gitlab-org/memory-team/team-tasks/issues/9).

### CODE REVIEW

File: gitlab/fetcher.go [hunk 1]
```diff
@@ -2,6 +2,9 @@ package gitlab
 
 import "github.com/xanzy/go-gitlab"
 
+// ReProjectName represents regular expression matching to a project name
+const ReProjectName = "^([^/]+(?:/[^/]+)+)"
```
Comment (sue445) [inline gitlab/fetcher.go]: I prefer to `reProjectName` (private) than `ReProjectName` (public).

Because this is used in only `gitlab` package.

Comment (hiroponz) [reply gitlab/fetcher.go]: It makes sense.

Fixed at 33b504a93654c0756c9585efa260da90205d8d17.

**Comment by hiroponz** [comment]
@sue445 I have added a test case. Do I need other test cases?

**Comment by sue445** [comment]
LGTM 🙆

**Comment by hiroponz** [comment]
Thank you 😄

**Comment by sue445** [comment]
Release v0.7.0
https://github.com/sue445/gitpanda/releases/tag/v0.7.0

**Pull request merged**